### ISSUE TITLE
ChromeDriver Click Not Working

### ISSUE THREAD

**Issue by mogulman52** [florentbr/SeleniumBasic#99]
Setup
Windows 7
Excel 2010
Seleniumbasic - Pulled 1 week ago

The Click method doesn't work in the ChromeDriver.  It works in Firefox and IE.  I tried executing Javascript to click it and it doesn't work.  I pulled the latest Chrome driver from the Selenium site and replacing the Seleniumbasic driver and it doesn't work.  This is a simple test case.

`Sub teste3()
   Dim strCaptcha As String
   Dim SubmitButton As WebElement
   Dim InputBox As WebElement
   Dim Element As WebElement

  Dim driver As ChromeDriver   '***\* Driver doesn't click button.
  Set driver = New ChromeDriver
  'Dim driver As IEDriver        '***\* Works on IE
  'Set driver = New IEDriver
  'Dim driver As FirefoxDriver    '***\* Works on Firefox
  'Set driver = New FirefoxDriver

  driver.Get "http://www.anp.gov.br/preco/prc/Resumo_Semanal_Index.asp"

  strCaptcha = driver.FindElementById("letra1").Text
  strCaptcha = strCaptcha & driver.FindElementById("letra2").Text
  strCaptcha = strCaptcha & driver.FindElementById("letra3").Text
  strCaptcha = strCaptcha & driver.FindElementById("letra4").Text

  Set InputBox = driver.FindElementById("txtValor")
  InputBox.SendKeys (strCaptcha)

  Set SubmitButton = driver.FindElementById("image1")

  SubmitButton.Click

  'rtn = driver.ExecuteScript("arguments[0].click();", SubmitButton)  'Tried Javascript approach - Chrome NO , Firefox YES

  driver.Wait (2000)

  For Each Element In driver.FindElementsByCss(".table_padrao tr td")
      Debug.Print Element.Text
  Next Element

End Sub

`

**Comment by mogulman52** [comment]
If you issue 2 consecutive click events Chrome seems to work.  I would still like the issue addressed but this is certainly a viable workaround.

**Comment by mogulman52** [comment]
This issue seems to be specific to one site and is not a general problem.

**Comment by balajisekar** [comment]
Yes, the problem is in site. An unknown script or frame or form or something is making problem and that not allowing to make click in automation or in manual.
